contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1260,19 +1292,21 @@ where:
 <variable_2>:: Specifies the explanation of the first variable.
 ....
 +
-Be sure to introduce the description list with "where:" and start each variable
-description with "Specifies."
+Be sure to introduce the description list with "where:" and start each variable description with "Specifies."
+
+[[backslash-for-long-code-line]]
+==== Backslash for long lines of code
```

Comment:
I wonder if it makes sense to move this ahead of `Callouts in code blocks`, since the callouts examples include the line break with annotation format (` \// (1)`). Putting the line break first would build the complexity more gradually :)